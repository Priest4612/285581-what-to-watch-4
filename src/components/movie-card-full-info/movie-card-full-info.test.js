import React from 'react';
import renderer from 'react-test-renderer';

import MovieCardFullInfo from './movie-card-full-info.jsx';


import filmsMock from '../../mocks/test-films-mock.json';


it(`Render MovieCardFullInfo`, () => {
  const {
    name, posterImage, description, rating, scoresCount,
    director, starring,
  } = filmsMock[1];
  const tree = renderer
    .create(<MovieCardFullInfo
      name={name}
      posterImage={posterImage}
      description={description}
      rating={rating}
      scoresCount={scoresCount}
      director={director}
      starring={starring}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
