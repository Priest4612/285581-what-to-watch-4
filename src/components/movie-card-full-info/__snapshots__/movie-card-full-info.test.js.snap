// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCardFullInfo 1`] = `
<div
  className="movie-card__wrap movie-card__translate-top"
>
  <div
    className="movie-card__info"
  >
    <div
      className="movie-card__poster movie-card__poster--big"
    >
      <img
        alt="Macbeth"
        height="327"
        src="img/macbeth.jpg"
        width="218"
      />
    </div>
    <div
      className="movie-card__desc"
    >
      <nav
        className="movie-nav movie-card__nav"
      >
        <ul
          className="movie-nav__list"
        >
          <li
            className="movie-nav__item movie-nav__item--active"
          >
            <a
              className="movie-nav__link"
              href="#"
            >
              Overview
            </a>
          </li>
          <li
            className="movie-nav__item false"
          >
            <a
              className="movie-nav__link"
              href="#"
            >
              Details
            </a>
          </li>
          <li
            className="movie-nav__item false"
          >
            <a
              className="movie-nav__link"
              href="#"
            >
              Reviews
            </a>
          </li>
        </ul>
      </nav>
      <div
        className="movie-rating"
      >
        <div
          className="movie-rating__score"
        >
          6.6
        </div>
        <p
          className="movie-rating__meta"
        >
          <span
            className="movie-rating__level"
          >
            Good
          </span>
          <span
            className="movie-rating__count"
          >
            901 ratings
          </span>
        </p>
      </div>
      <div
        className="movie-card__text"
      >
        <p>
          https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm
        </p>
        <p
          className="movie-card__director"
        >
          <strong>
            Director: Я нёс цветы, дед свою поломанную ветку, мы шли молча.
          </strong>
        </p>
        <p
          className="movie-card__starring"
        >
          <strong>
            Starring: Дэвид Линч, Альфред Хичкок, Сидни Люмет, Квентин Тарантино, Майкл Манн, Кристофер Нолан, Леонид Гайдай, Билли Уайлдер
          </strong>
        </p>
      </div>
    </div>
  </div>
</div>
`;
